c_glib/arrow-glib/basic-array.cpp

```diff
@@ -544,6 +544,34 @@ garrow_array_view(GArrowArray *array,
   }
 }
 
+/**
+ * garrow_array_diff_unified:
+ * @array: A #GArrowArray.
+ * @other_array: A #GArrowArray to be compared.
+ *
+ * Returns: (nullable) (transfer full): The string representation of
+ *   the difference between two arrays as unified format.
+ *
+ *   It should be freed with g_free() when no longer needed.
```

Comment:
Could you add a description for `NULL` case?

Reply:
I've added a description.